opentimelineio/core/item.py

```diff
@@ -101,6 +101,13 @@ def trimmed_range(self):
 
         return self.available_range()
 
+    def visible_range(self):
```

Comment:
Great! 

However would it make sense to also have clear `visible_range()` and `visible_range_in_parent()`? This seems to return one or the other depending on if there is a parent or not. 

Lets say we have a track with "[Head_Leader, Clip_A, 24frame_Transition, Clip_B]" (A and B being 100 frame clips). I could see wanting to ask Clip_B for its `visible_range()` (Lets say 101-200), `trimmed_range()` (113-200),  `visible_range_in_parent()` (268-367), and `trimmed_range_in_parent()` (280-367).. (I could totally be off by one or so in those numbers somewhere). 

As to what to return when there is no parent? `trimmed_range()` seems fine.

Reply:
Yeah, I see what you're getting at. I'm planning to add an "Anatomy of a Clip" section to the wiki diagrams that we just updated which should address this directly. The differences between all these *_range() methods are confusing without pictures, which is kind of a bug in the API.